.pre-commit-config.yaml

```
@@ -14,6 +14,21 @@
 # limitations under the License.
 
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: 'v5.0.0'
+    hooks:
+      - id: end-of-file-fixer
+        files: \.(mps|json|yaml|yml|txt)$
+        exclude: ^datasets/.*\.(mps|json|yaml|yml|txt)$
+      - id: trailing-whitespace
+        files: \.(mps|json|yaml|yml|txt)$
+        exclude: ^datasets/.*\.(mps|json|yaml|yml|txt)$
+      - id: check-builtin-literals
+        files: \.(mps|json|yaml|yml|txt)$
+        exclude: ^datasets/.*\.(mps|json|yaml|yml|txt)$
```

```suggestion
      - id: check-builtin-literals
```

`check-builtin-literals` should only be looking at Python files: https://github.com/pre-commit/pre-commit-hooks/blob/582b9c66e0791980685c8b80bb11e6a3df1a06ef/.pre-commit-hooks.yaml#L25

So I think this filtering is unnecessary and should be removed.